docs-new/docs/contribute/docs/spell-check.md

```diff
@@ -0,0 +1,76 @@
+# Spell Checker
+
+All PRs that are pushed to a Keptn repository
+are run through a spell checker that is based on the
+[check-spelling](https://github.com/check-spelling/check-spelling)
+GitHub action.
+
+Note, however, that you are still responsible for reading your text carefully.
+The tool does not flag if you used a real word that is not the right word,
+if you misuse "setup" versus "set up", and so forth.
+
+## Handling Spell Checker errors
+
+If you get a Spell Checker error:
+
+1. Right-click the `details` link for Spell Checker
```

Comment:
why right click? this just opens the context menu on most OSes.
if anything, I think it should be left click, but we can also just put "click"